### PR TITLE
docs: update README.md

### PR DESCRIPTION
**Pull request by Pythonberg1997**
### Description

This pr is to update README.md

### Rationale

Fix typo and update description

### Changes

no notable changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,14 +125,14 @@ bash ./deployment/localup/localup.sh all 1 7
 
 You can use the following approach if you do not know how. 
 
-Run following codes in Greenfield e2e tests to get validator and challenger private keys
-```go
-// please refer to greenfield repo for more information
-// challenger key
-fmt.Println(common.Bytes2Hex(s.Challenger.GetPrivKey().Bytes()))
+```shell
+$ ./build/bin/gnfd keys export validator_bls0 --unsafe --unarmored-hex --home=./deployment/localup/.local/validator0 --keyring-backend=test
```
Comment (guagualvcha) [inline README.md:129]: can you change this `/deployment/localup/.local/validator0` to a geeneral one? so is `validator_bls0`

File: README.md [hunk 2]
```diff
@@ -33,67 +32,70 @@ This off-chain application comprises 7 main goroutines: Monitor, Verifier, Vote
...
-  "aws_region": set this if you chose "aws_private_key"
-  "aws_secret_name": set this if you chose "aws_private_key"
-  "username": set this if you chose "local_private_key"
-  "password": set this if you chose "local_private_key"
-  "max_idle_conns": 20, (set according to your db performance)
-  "max_open_conns": 40, (set according to your db performance)
-  "debug_mode": false  
-}
-```
+1. Set your private key import method (via file or aws secret), deployment environment and gas limit.
```
Comment (guagualvcha) [inline README.md:36]: can you guide user to refer to `https://github.com/bnb-chain/bnb-chain-charts/blob/d09a93d7b18e0f2afb537a23276a95c877e070b6/gnfd-challenger-testnet-values/values.yaml#L8` as well? 
As it is the testnet config.

File: README.md (hunk 1 above)
Comment (Pythonberg1997) [reply README.md:129]: Done

File: README.md (hunk 2 above)
Comment (Pythonberg1997) [reply README.md:38]: Done

File: README.md [hunk 3]
```diff
@@ -33,67 +32,72 @@ This off-chain application comprises 7 main goroutines: Monitor, Verifier, Vote
...
-  "filename": "log.txt",
-  "max_file_size_in_mb": 100 (file size threshold)  
-  "max_backups_of_log_files": 2 (backup count threshold)
-  "max_age_to_retain_log_files_in_days": 10 (backup age threshold)
-  "use_console_logger": true,
-  "use_file_logger": false,
-  "compress": false
-}
-```
+See [config.json](https://github.com/bnb-chain/bnb-chain-charts/blob/main/gnfd-challenger-testnet-values/values.yaml#L8). Reference for a complete testnet config file.
```
Comment (forcodedancing) [inline README.md:36]: Maybe, we should also have a section to let the users know 1) who should run it - only validators, 2) why they need to run it, and 3) the benefit of running it - reward.

**Review by guagualvcha — APPROVED** [review]

**Pull request merged**